web/src/user_topic_popover.ts

```diff
@@ -28,24 +32,30 @@ export function initialize() {
             popover_menus.popover_instances.change_visibility_policy = instance;
             popover_menus.on_show_prep(instance);
             const $elt = $(instance.reference).closest(".change_visibility_policy").expectOne();
-            const stream_id = $elt.attr("data-stream-id");
-            const topic_name = $elt.attr("data-topic-name");
+            const stream_id_str = $elt.attr("data-stream-id");
+            const topic_name_str = $elt.attr("data-topic-name");
             $elt.addClass("visibility-policy-popover-visible");
+            assert(stream_id_str !== undefined);
+            assert(topic_name_str !== undefined);
+
+            stream_id = Number.parseInt(stream_id_str, 10);
+            topic_name = topic_name_str;
 
-            instance.context =
```

Comment:
Removing this hunk can be a prep-commit. You can learn about our commit discipline from [Zulip docs](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#).